### ISSUE TITLE
What exactly are the charging protocol of cells #91, #100, and #124?

### ISSUE THREAD

**Issue by WenPengfei0823** [rdbraatz/data-driven-prediction-of-battery-cycle-life-before-capacity-degradation#32]
Thanks for your contributions. The charging protocol of cells #91, #100, and #124 is marked as "4.36C(80%)-4.36C" in the supplementary document but "4.8C(80%)-4.8C" in the loaded data. Which one is the exact protocol?

**Comment by JazerJu** [comment]
I agree with you that the charging protocol of cells #91, #100, and #124 are dfferent in supplementary document and loaded data,And we don't have exact channel id in the .mat file. However ,I reckon that doesn't matter cuz it's just a pedal on the way forward to give a prediction for batteries in my lab
